katpoint/test/test_delay.py

```diff
@@ -126,10 +126,10 @@ def test_offset(self):
         delay1, phase1 = self.delays.corrections(target3, self.ts,
                                                  self.ts + 1.0, offset)
         # Conspire to return to special target1
-        self.assertEqual(delay0['A2h'], extra_delay, 'Delay for ant2h should be zero')
-        self.assertEqual(delay0['A2v'], extra_delay, 'Delay for ant2v should be zero')
-        self.assertEqual(delay1['A2h'][0], extra_delay, 'Delay for ant2h should be zero')
-        self.assertEqual(delay1['A2v'][0], extra_delay, 'Delay for ant2v should be zero')
+        np.testing.assert_almost_equal(delay0['A2h'], extra_delay, decimal=15)
+        np.testing.assert_almost_equal(delay0['A2v'], extra_delay, decimal=15)
+        np.testing.assert_almost_equal(delay1['A2h'][0], extra_delay, decimal=15)
+        np.testing.assert_almost_equal(delay1['A2v'][0], extra_delay, decimal=15)
```

Comment:
Docs for assert_almost_equal says
> It is recommended to use one of `assert_allclose`, `assert_array_almost_equal_nulp` or `assert_array_max_ulp` instead of this function for more consistent floating point comparisons.

Reply:
This is old katpoint, which is riddled with `np.testing.assert_almost_equal`. New katpoint fixed all of that. I'll fix it when I port it to the new code.